tmk_core/common/action_tapping.c

```diff
@@ -24,12 +24,14 @@
 #    define IS_TAPPING_RECORD(r) (IS_TAPPING() && KEYEQ(tapping_key.event.key, (r->event.key)) && tapping_key.keycode == r->keycode)
 #endif
 
-__attribute__((weak)) uint16_t get_tapping_term(uint16_t keycode, keyrecord_t *record) { return TAPPING_TERM; }
+uint16_t tapping_term = TAPPING_TERM;
+
+__attribute__((weak)) uint16_t get_tapping_term(uint16_t keycode, keyrecord_t *record) { return tapping_term; }
```

Comment:
given this doesnt work if you override `get_tapping_term` it might be cleaner to implement its own non weak copy of get_tapping_term within `quantum/process_keycode/process_tap_term_keys.c`